railties/lib/rails/generators/rails/plugin/templates/Rakefile

```diff
@@ -25,5 +25,5 @@ load 'rails/tasks/statistics.rake'
 
 <% unless options[:skip_gemspec] -%>
 
-Bundler::GemHelper.install_tasks
+require 'bundler/gem_tasks'
```

Comment:
It is for that to add the `rake clean` task and `rake clobber` task. 
Ref: [include 'pkg/' in rake/clean's CLOBBER array by jasonkarns · Pull Request #4022 · bundler/bundler](https://github.com/bundler/bundler/pull/4022) 

It is as it will help to remove pkg, I think that than may be added.